src/webparts/controlsTest/components/ControlsTest.tsx

```diff
@@ -1,20 +1,19 @@
 import * as React from "react";
 
-import {
```

Comment:
@joaojmendes this file is massively modified and really hard to track real changes. I believe most changes were made by some VS Code extension to format the file. This makes it hard to review so any chance you could revert the formatting changes?

Reply:
rollback to previous version os controltest